CMakeLists.txt

```diff
@@ -52,11 +52,11 @@ option(ENABLE_GCOV "gcov code coverage feature" OFF)
 option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
-option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
```

Comment:
@yang1070 
No. It is obsolete option because there is another place to set that
https://github.com/LuxoftAKutsan/sdl_core/blob/06e124fca7ca23cb19b7586f5cd8e14c6878b9d9/CMakeLists.txt#L61